Analytics/Configuration/Transaction.ts

```diff
@@ -1,105 +1,106 @@
+// 		risk: { selector: "value.risk", transform: "number" },
+// 		state: { selector: "value.state", transform: "stringify" },
+// 		// eslint-disable-next-line
+// 	} as const satisfies filter.Mapping.RecordWithSelector<string>;
+// 	export type Fields = keyof typeof mapping
+// 	export const schema: listener.BigQueryApi.BaseField<Fields>[] = [
+// 		{ name: "counterpartType", type: "STRING" },
+// 		{ name: "counterpart", type: "STRING" },
+// 		{ name: "counterpartCode", type: "STRING", mode: "NULLABLE" },
+// 		{ name: "currency", type: "STRING" },
+// 		{ name: "amount", type: "NUMERIC" },
+// 		{ name: "description", type: "STRING" },
+// 		{ name: "organization", type: "STRING" },
+// 		{ name: "accountId", type: "STRING" },
+// 		{ name: "accountName", type: "STRING", mode: "NULLABLE" },
+// 		{ name: "account", type: "STRING" },
+// 		{ name: "accountType", type: "STRING" },
+// 		{ name: "type", type: "STRING", mode: "NULLABLE" },
+// 		{ name: "direction", type: "STRING", mode: "NULLABLE" },
+// 		{ name: "id", type: "STRING" },
+// 		{ name: "referenceSupplier", type: "STRING", mode: "NULLABLE" },
+// 		{ name: "referenceReference", type: "STRING", mode: "NULLABLE" },
+// 		{ name: "referenceReturnId", type: "STRING", mode: "NULLABLE" },
+// 		{ name: "referenceEndToEndId", type: "STRING", mode: "NULLABLE" },
+// 		{ name: "posted", type: "DATETIME" },
+// 		{ name: "transacted", type: "DATETIME", mode: "NULLABLE" },
+// 		{ name: "by", type: "STRING", mode: "NULLABLE" },
+// 		{ name: "actualBalance", type: "NUMERIC" },
+// 		{ name: "reservedBalance", type: "NUMERIC" },
+// 		{ name: "availableBalance", type: "NUMERIC" },
+// 		{ name: "operations", type: "RECORD", mode: "REPEATED", fields: [
+// 				{ name: "account", type: "STRING" },
+// 				{ name: "currency", type: "STRING" },
+// 				{ name: "changes", type: "RECORD", mode: "REPEATED", fields: [
+// 					{ name: "key", type: "STRING" },
+// 					{ name: "value", type: "RECORD", fields: [
+// 						{ name: "type", type: "STRING" },
+// 						{ name: "amount", type: "NUMERIC" },
+// 						{ name: "status", type: "STRING" },
+// 						{ name: "result", type: "STRING", mode: "NULLABLE" },
+// 					]}
+// 				] },
+// 				{ name: "type", type: "STRING" },
+// 				{ name: "transaction", type: "STRING" },
+// 				{ name: "counter", type: "NUMERIC" },
+// 				{ name: "created", type: "DATETIME" },
+// 				{ name: "signature", type: "STRING" },
+// 				{ name: "previous", type: "STRING" },
+// 			]
+// 		},
+// 		{ name: "status", type: "STRING" },
+// 		{ name: "rail", type: "STRING", mode: "NULLABLE" },
+// 		{ name: "railType", type: "STRING" },
+// 		{ name: "flags", type: "STRING", mode: "REPEATED" },
+// 		{ name: "oldFlags", type: "STRING", mode: "REPEATED" },
+// 		{ name: "notes", type: "STRING", mode: "REPEATED" },
+// 		{ name: "risk", type: "NUMERIC", mode: "NULLABLE" },
+// 		{ name: "state", type: "STRING", mode: "NULLABLE" },
+// 	]
+// }
```

Comment:
Why the comment?

Reply:
Currently the updated model lives in worker-banking-analytics, but I could move it here I suppose